test/receiver/receiver.go

```diff
@@ -65,13 +66,42 @@ func main() {
 	outDir := flag.String("outdir", "", "Output directory. Absolute path. Optional.")
 	profiler := flag.String("profiler", "", "Where should the profiler listen?")
 	exitAfter := flag.Duration("exitAfter", time.Second*10, "Exit after not receiving any message for this time. Will work only of outDir == ''. Will not exit if Duration is 0")
+	localAPIPort := flag.Int64("local-api-port", 8024, "specify which port the local HTTP API should be listening on")
```

Comment:
Why is this the default port? Wouldn't it make more sense to make 0 the default and only start a server when it's set to non-zero?